src/styles/header.module.css

```diff
@@ -0,0 +1,47 @@
+.header_container{
+  background-color: var(--header-footer); 
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.main_nav{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight:normal;
+  text-decoration: none;
+  font-size: 1rem;
+  width: 100%;
+  margin-inline: 5%;
+}
+
+.nav_left{
+  list-style: none;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+/* NOTE:  z jakiegoś powodu to nie działa, gdy link do czegoś prowadzi */
```

Comment:
ditto: english, but solution is here:  https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_links